Scrumdinger/Views/MeetingView.swift

```diff
@@ -48,14 +49,20 @@ struct MeetingView: View {
                 let newHistory = History(attendees: scrum.attendees,
                                          lengthInMinutes: scrumTimer.secondsElapsed / 60,
                                          transcript: transcript)
-                scrum.history.insert(newHistory, at: 0)
+                do {
+                    try Realm().write {
+                        scrum.historyList.insert(newHistory, at: 0)
```

Comment:
instead of relying on the insert position for displaying history why not retrieve the `historyList` sorted by date?

Reply:
Could work either way, but I want to minimise the changes between the original tutorial and the Realm version